.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
What are your thoughts with including the `.travis.yml`?

Reply:
It is the standard for Ansible roles included in Ansible Galaxy, and generated as part of the skeleton with `ansible-galaxy init`. I figured it wouldn't hurt anything to leave it there, even though we won't use it.

Reply:
sounds like a plan